### DISCUSSION TITLE
Reaction button state is almost impossible to distinguish

### DISCUSSION THREAD

**Discussion by ericsampson** [community/community#22202] [Accessibility]
<p>In the screenshot below, I have clicked on the top reaction buttom but not the bottom reaction button.<br>
However, this difference in state is virtually impossible to distinguish - the selected one has the barest of ultradark blue hue if you get your eyes right up to the screen. It would be nice if the selection state was a little more obvious. Thanks!!<br>
Generally I’m loving dark mode, other than the often mentioned items of making changed words in diffs pop out more, and making a ‘dim’ theme option vs the current black.</p>
<p><img src="https://user-images.githubusercontent.com/1183853/181069482-8f0c9e92-8ef4-4146-b12f-48e62c03f4f4.png" alt="image" data-base62-sha1="xvs6aItaohwnjqXrBWfKseN3aqa" width="203" height="340"></p>

**Comment by gaknoll** [comment, marked as answer]
<p>Thanks <a class="mention" href="/ericsampson">@ericsampson</a> for this feedback! I agree that the contrast is low which makes it hard to differentiate between the default and activated states. I’ve added this to our design backlog to review and shared it with the team!</p>

**Comment by cehfisher** [comment]
Hi @ericsampson - I wanted to follow-up on this stale issue and ask if the dark mode theme changes have resolved this issue for you? If yes, could you close this issue as resolved? If no, what additional changes are needed? (cc @queenofcorgis)
